pdfstitch

```diff
@@ -41,14 +42,17 @@ GetOptions
 (
     'h|help' => \$help,
     'g|genmeta' => \$genmeta,
+    'dc|defaultcrop=s' => \$defaultcrop,
```

Comment:
"dc" will not work for short options. This has to be a single character. I'd suggest 'd'.